### PR TITLE
Update FFmpeg for Rockchip

### PR DESCRIPTION
**Pull request by MarcA711**
Changes:
- update FFmpeg
- update docs
- remove go2rtc presets for older FFmpeg version

ToDo/ known issues:
- reimplement hardware object detection; I want to use yolonas as model, but I am not sure if there will be new license issues, does anybody have experience with this?
- include hardware acceleration presets for go2rtc (see https://github.com/AlexxIT/go2rtc/issues/979)
- there is a hardware limitation that can lead to crashes (see https://github.com/rockchip-linux/mpp/issues/560)

### CODE REVIEW

**Review by NickM-27 — APPROVED** [review]

**Comment by NickM-27** [comment]
@blakeblackshear may be able to provide more info on the first question

**Comment by blakeblackshear** [comment]
I believe yolo-nas should be fine. I don't think they interpret the AGPL license quite the same way. I will need to train from scratch to use it in Frigate+, but I think it can be used here. You may want to ask in the YOLO NAS repo directly though before doing it.

**Comment by MarcA711** [comment]
Hey @blakeblackshear,
unfortunately, I did not get a conclusive answer if I can use yolo-nas with pre-trained weights (see https://github.com/Deci-AI/super-gradients/issues/1993). Do you think it is okay to use the pre-trained weigths as long as they don't object?
